app/src/main/kotlin/com/br/myfavoritehero/base/BaseViewModel.kt

```diff
@@ -41,11 +42,20 @@ abstract class BaseViewModel : ViewModel(), LifecycleObserver {
                     HttpURLConnection.HTTP_BAD_METHOD -> {
                         errorResponse.extra = R.string.unknow_error
```

Comment:
This conditional is needed? else condition handle this....

Reply:
HttpURLConnection.HTTP_CONFLICT
                    HttpURLConnection.HTTP_NOT_FOUND
                    HttpURLConnection.HTTP_BAD_METHOD

Reply:
Yes, the server can change the type request and the app should handle that.

Reply:
I mean the else condition makes the same action of this ones:
                HttpURLConnection.HTTP_CONFLICT
                HttpURLConnection.HTTP_NOT_FOUND
                HttpURLConnection.HTTP_BAD_METHOD
if we don't do different action for each case this conditions don't make sense :)